spec/models/discourse_connect_spec.rb

```diff
@@ -904,7 +904,7 @@ def test_parsed(parsed, sso)
       user = sso.lookup_or_create_user(ip_address)
       expect(user.active).to eq(true)
 
-      user.primary_email.update_columns(email: "xXx@themovie.com")
```

Comment:
This update wouldn't get picked up in the relevant code path when using `#update_columns` instead of `#update`.